.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+    branches:
+      # Branches from forks have the form 'user:branch-name' so we only run
+      # this job on pull_request events for branches that look like fork
+      # branches. Without this we would end up running this job twice for non
+      # forked PRs, once for the push and then once for opening the PR.
+    - '**:**'
+
+jobs:
+  build:
+    name: "Build"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+#      - name: Cache Gradle
+#        uses: actions/cache@v2
+#        with:
+#          path: |
+#            ~/.gradle/wrapper
+#            ~/.gradle/caches
+#          key: ${{ runner.os }}-${{ hashFiles('**/lockfiles') }}
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - id: get_version
+        name: "Get Version"
+        uses: battila7/get-version-action@v2
+      - name: Build with Gradle
+        if: startsWith(github.ref, 'refs/tags/v') != true
```

Comment:
Why this `if`? We always want to check. Then you can remove the redundant check in the next step.

Reply:
how can i otherwise split the release/non release tag.
i have zero knowledge with github actions
and this is what i have found
do you have and better idea?

i have no idea if steps can see each others results

Reply:
thus i have splitted the thing into two scriots

- normal builds
- tags builds

Reply:
I don't understand what you mean by `how can i otherwise split the release/non release tag.`. This step (the check step) needs not `if`. We always want to check. Only the release needs an `if`.

Reply:
as i cant test the release i dont know if it will work if i do

(1) step: gradle check
(2) step gradle release

thus i did

(1) on non tags: gradle check
(2) on tags: gradle check && gradle release

Reply:
There's no difference between the two.

Reply:
?

```
        run: |
          ./gradlew check
```

vs

```
        run: |
          ./gradlew check
          ./gradlew release "-PreleaseVersion=${{env.TAG_NAME}}"
```

Reply:
You always run the `check` step, no matter if tag or not. That's what I meant with remove the "if" on that step.

Reply:
@oehme please take over this PR
o have no time anymore.
have spent half the day to get the crap as it is now

Reply:
@cdietrich please stop using words like "crap", "shit", ??????????? etc. It's unconstructive and gets everyone else in a bad mood. I'm sorry you're having a bad day. I'll take over from here.

Reply:
i am sorry for my tone. and thank for taking over
but eachand every time i work on anyting xtext + gradle related my mood worsens a lot. so it turn a decent day into a bad and a bad day into a terrible day.
the only consequence is: consider xtext+gradle as discontinued and act if it was no longer existing

Reply:
I can do some minimal maintenance where necessary and maybe others will step up. Don't feel obligated to do something you don't like.